static/swagger/zulip.yaml

```diff
@@ -541,6 +540,92 @@ paths:
                   items:
                     $ref: '#/definitions/stream'
 
+  /streams/{stream_id}:
+    patch:
+      description: Update a stream's name, description or privacy setting.
+      operationId: zerver.views.streams.update_stream_backend
+      parameters:
+      - name: stream_id
+        in: path
+        description: The ID of the stream to modify.
+        required: true
+        type: integer
+      - name: new_name
+        in: formData
+        description: |
+                     New name for the stream.
+
+                     Must be a valid JSON string, so it has to be enclosed in
```

Comment:
Maybe "Must be a valid JSON string (enclosed in double quotes and special characters escaped with backslashes"?